src/geocoder.js

```diff
@@ -33,11 +33,8 @@ export default class Geocoder {
     this.lastGeocode = new Date();
     this.currentRequests = 0;
 
-    this.cache = new Cache(options.cacheFile);
-    this.googlemaps = googlemaps;
-
-    this.googlemaps.config('google-client-id', options.clientId);
-    this.googlemaps.config('google-private-key', options.privateKey);
+    this.cache = new GeoCache(options.cacheFile);
+    this.googlemaps = geoCoder.init(options);
```

Comment:
Either the var is indicating that it is a Google lib or not. Naming is inconsistent: googleMaps, geoCoder, googlemaps for the same thing.